docs/README-submodule-api.md

```diff
@@ -191,12 +191,16 @@ Stops the application
 
 Parameters:
 * `application_name` -- application name
+* `source_directory` -- directory containing the manifests. In case of k8s
+  it is a directory where we run apply, or docker-compose up in case of docker-compose.
 
 #### START
 Starts the application
 
 Parameters:
 * `application_name` -- application name
+* `source_directory` -- directory containing the manifests. In case of k8s
+  it is a directory where we run apply, or docker-compose up in case of docker-compose.
 
```

Comment:
Don't you think we will need to add also `source_directory` to  rollback command?

Reply:
yes. good catch. added.